### PR TITLE
Feature/request param

### PR DESCRIPTION
**Pull request by golichenko02**

### CODE REVIEW

**Review by OlehYakovenko — APPROVED** [review]

File: src/test/java/io/github/bobocodebreskul/demointegration/controller/RequestParamController.java
Comment (OlehYakovenko) [inline src/test/java/io/github/bobocodebreskul/demointegration/controller/RequestParamController.java:1]: Do we really need this Demo? I propose to delete

File: src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java [hunk 1]
```diff
@@ -359,4 +370,26 @@ private Object getBodyFromRequest(Class<?> bodyType, HttpServletRequest req) {
...
+    log.debug("Retrieving request parameter with name: {}", requestParamName);
+    return Optional.ofNullable(req.getParameter(requestParamName))
+        .map(value -> castValue(value, parameter.getType()))
+        .orElseGet(() -> {
+          log.warn("Cannot find request parameter [{}] in request", requestParamName);
+          return null;
+        });
+  }
+
+  private void validateRequestParameterType(Class<?> type) {
```
Comment (s-barabash) [inline src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java]: Let's move it to separate class

File: src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java [hunk 2]
```diff
@@ -321,7 +332,7 @@ private boolean isHttpResponse(Parameter parameter) {
 
   private void validateRequestMethod(HttpServletRequest req) {
```
Comment (s-barabash) [inline src/main/java/io/github/bobocodebreskul/server/DispatcherServlet.java]: Let's move to separate class

**Review by AlexandrKarpachov — APPROVED** [review]

File: src/test/java/io/github/bobocodebreskul/demointegration/controller/RequestParamController.java
Comment (golichenko02) [reply src/test/java/io/github/bobocodebreskul/demointegration/controller/RequestParamController.java:1]: It is used for tests

**Review by OlehYakovenko — APPROVED** [review]

**Pull request merged**